Add Professional email setup card checklist to WordPress dashboard

AllTerrainDeveloper opened the pull request:
#### Changes proposed in this Pull Request

Add task to complete Professional email setup
![image](https://user-images.githubusercontent.com/5689927/127175102-df265bca-36ec-4e27-8938-0237a713183a.png)


#### Testing instructions

1. Have a new site created from the onboarding steps
2. Go to dashboard
3. Check tasks lists
4. One of the tasks to complete should be "Setup your professional Email"
5. Complete the setup -> The task should be marked as completed.

Related to {1200182182542585-as-1200588428897562}

AllTerrainDeveloper committed:
Emails: Add stub for Professional email task completion in dashboard

AllTerrainDeveloper committed:
Emails: Add stub for Professional email task completion in dashboard

AllTerrainDeveloper committed:
Emails: Align tasks const with PHP development

AllTerrainDeveloper committed:
Emails: Align tasks const with PHP development

andres-blanco committed:
Theme Showcase: Fix alignment of search autocomplete (#54905)

* Fix alignment of search autocomplete

a8ci18n commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +191,18 @@ export const getTask = (
+				description: translate(
+					'Complete your Professional Email setup to start sending and receiving emails from your custom domain today.'
+				),
+				actionText: translate( 'Complete Setup' ),
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Complete setup' )` *ES Score: 10*

macbre committed:
Backup download: show file size (#54174)

E.g. http://calypso.localhost:3000/backup/macbretest.wpcomstaging.com/download/1624906075.02

This change will use a new "bytesFormatted" API response entry introduced by https://code.a8c.com/D63518

jwebbdev committed:
Added Backup Plugin to Backup Minimum Plugin reqs (#54899)

rcanepa committed:
Jetpack site-less Checkout: enable feature flag in production environments (#54931)

kwight committed:
E2E: Update the configuration file. (#54897)

rafaelgallani committed:
Domains: Remove duplicate "organization" text input in the contact information section on checkout (#54903)

* Remove duplicate "organization" field

* Add both 'click' and 'toggle' to HiddenInput state

* Remove lodash 'get'

* Remove unused code

noahtallen committed:
Plugin Builds: Merge trunk before running build (#54809)

The problem with building with only the branch is that
the comparision between the current branch build and
the latest trunk build is out of date. As a result, your
stale branch can very easily be considered "different"
from the latest release build, even though the final,
merged branch would not actually change it. In other
words, it's different because it's out of date.

This solves the problem by merging trunk first. As a
result, the comparison with the trunk build will never
be stale, as the latest changes are always included.

This will not affect release builds.

The main downside is that the final build artifact in
branches will not exactly match your local development
environment.

fullofcaffeine committed:
[ETK/error-reporting] Activate sentry for a segment of users  (#54257)

* Activate sentry for a segment of users, and a12s if the blog has the `error-reporting-use-sentry` sticker

* Add debug output

* Improve how data is passed from php and handled in the client js

* Simplify expression

* Add test for the error reporter activation logic

* Fix test name and improve comment

* Fix stubbing for `is_automattician`

The previous implementation created a chicken-and-egg situation with the test env and broke the logic in production.

In the ETK PHP test env, `is_automattician` is not available globally. At first, I defined it under the `A8C\FSE\ErrorReporting` namespace in the test. This worked for the stubbing purposes, but required me to change the `function_exists` call in the implementation to include the namespace in order for it to work as expected in the test env. I thought that in production, this check would work, too, but it didn't. The function is defined globally in production and the check for it under that namespace failed, meaning that the `should_activate_sentry` always returned `false` in production.

The solution was to remove that `function_exists` check which was creating more trouble than it's worth, and make sure the function was available in the test env, for all tests.

* Change to the definitive Sentry DSN

* Change initial user test segment to 1%

* Dedup packages

Using `npx yarn-deduplicate`.

 # Changesto be committed:

* Capture head errors for Sentry too

* Add debug code for head errors, will be reverted before merge

* Improve error head capture logic for Sentry

Sentry.captureException doesn't return a Promise so it doesn't converge to the same type in order to have a single `reportError` that is awaited by `Promise.allSettled`. Sentry requires `Sentry.flush()` to be called if you want to guarantee that all events are pushed. This in turns returns a Promise. Better to just have separate logic for each error reporter, in this case.

* Make the name of the obj used to pass data from PHP to JS more unique

It's in window and it was a bit too general, so another plugin could maybe add something that could match and replace it (or it could affect an existing reference).

* FSE -> ETK

* Remove debug code

* Spelling fix

* Set the priority of the `admin_print_scripts` action to 0 to make sure it is added as early as possible

worldomonation committed:
Playwright: restructure the `support` spec. (#54826)

* Restructure the `wp-support__search-spec`.

- remove usage of SupportArticleComponent.
- combine two cases together to reduce repetitive processes.
- add new test step to visit the support article.

* Further combine test suites to reduce repetitive actions.
- combine the two invalid search strings into one.

* Parametrize each support tests to run on both Simple and Atomic.

* Additional step to clear the keyword to work around #54935 in mobile version of the test.

DanielSmithMichigan committed:
Themes Showcase: Removed theme options from "InstallThemeButton" (#54855)

Co-authored-by: danielsmithmichigan <danielsmithmichigan@users.noreply.github.com>

elliottprogrammer committed:
Make form submit with Enter key pressed. (#54928)

blackjackkent committed:
Fix padding and line height on theme showcase tabs (#54937)

blackjackkent committed:
Fix logic so 'All' tab highlights correctly on theme search card (#54910)

worldomonation committed:
Move several GitHub issue templates to YAML based template. (#54728)

* Move flaky e2e report.

* Move simple-atomic parity template.

* Move tooling request

* Better formatting

* Formatting fixes

* Fix typo

IanRamosC committed:
Partner Portal: Add the 'credit-card' redux store (#54570)

* Partner Portal: Add the 'credit-card' redux store

This PR adds the base store for the 'credit-card' payment method store that is going to be used in the Partner Portal. It's pretty much a port from what's currently used in the composite-checkout package and might need improvements later.

This is going to be used behind a feature flag, so it won't harm having this merged

* Partner Portal: Add initial state for payment method

* Partner Portal: Address the latest feedback

This commit removes unnecessary code, and improves the type definitions and readability for the `payment-methods` store.

* Partner Portal: Add more context to isTouched description comment

arthur791004 committed:
Update copy in the growth summit card (#54876)

bluefuton committed:
Jetpack Search: improve handling of missing Jetpack site (#54634)

guarani committed:
Hide sidebar when Calypso is viewed from WC apps (#54189)

Hide sidebar when Calypso is viewed from WC apps

The WooCommerce apps embed Calypso to show specific parts of it, so have no need for the sidebar. This PR uses the browser's (or webview's) user agent to hide the sidebar when the WCiOS or WCAndroid user agent is present.

deBhal committed:
My Home: Move "Finish store setup" into Site setup checklist (#54148)

* Render woocommerce setup checklist task

* Update settings of woocommerce task

* Adjust styles of task timing

* Adjust timing of woo setup back to 7

Co-authored-by: arthur <arthur.chu@automattic.com>

fullofcaffeine committed:
Revert "[ETK/error-reporting] Activate sentry for a segment of users  (#54257)" (#54951)

This reverts commit ebd69ba8b998d797c8f07544ead510f06e136f87.

scinos committed:
build: Add action to create temporary calypso.live links (#54914)

* Add action to create temporary calypso.live links

* Fix title

* Try one line

* Fix the message

scinos committed:
chore: Refactor packages/design-picker to use import/order (#54915)

scinos committed:
chore: Refactor packages/browser-data-collector to use import/order (#54916)

scinos committed:
chore: Force base-styles resolution (#54927)

scinos committed:
chore: Remove unused files (#54831)

renovate[bot] committed:
fix(deps): update babel monorepo (#54877)

* fix(deps): update babel monorepo

* Undo bad merge

Co-authored-by: Renovate Bot <bot@renovateapp.com>
Co-authored-by: Sergio Cinos <sergio.cinos@gmail.com>

jsnajdr committed:
Add polyfill for Element.closest (#54923)

* Add polyfill for Element.closest

* Update yarn.lock

Co-authored-by: Sergio Cinos <sergio.cinos@gmail.com>

AllTerrainDeveloper committed:
Emails: Fix Google Workspace inline layout for upsell and add mailbox. (#54921)

tjcafferkey committed:
Quick Switch Feature Modal (#54687)

mmtr committed:
Nav Unification: Remove "show wp-admin pages" toggle (#54387)

flootr committed:
chore: migrate `boot` to `import/order` (#54893)

flootr committed:
chore: migrate `packages/i18n-utils` to `import/order` (#54888)

flootr committed:
chore: migrate `packages/viewport-react` to `import/order` (#54956)

flootr committed:
chore: migrate `packages/request-external-access` to `import/order` (#54957)

flootr committed:
chore: migrate `desktop/e2e` to `import/order` (#54955)

AllTerrainDeveloper committed:
Emails: Fix lint problems

AllTerrainDeveloper committed:
Emails: Fix lint problems

AllTerrainDeveloper committed:
Emails: Lint problems fix

AllTerrainDeveloper committed:
Emails: Lint problems fix

flootr committed:
chore: migrate `packages/babel-plugin-transform-wpcalypso-async` to `import/order` (#54958)

flootr committed:
chore: migrate `client/notifications` to `import/order` (#54959)

flootr committed:
chore: migrate `client/test-helpers` to `import/order` (#54961)

flootr committed:
chore: migrate `packages/i18n-calypso-cli` to `import/order` (#54962)

mattgawarecki committed:
Activity Log: Only show 'Share this event' for backups (#54904)

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +191,18 @@ export const getTask = (
+		case CHECKLIST_KNOWN_TASKS.PROFESSIONAL_EMAIL_MAILBOX_CREATED:
+			taskData = {
+				timing: 2,
+				title: translate( 'Setup your professional Email' ),
```
I think this should be `Set up your Professional Email`. We may want to supply `Professional Email` via a parameter, but that feels like overkill for this first pass.

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +191,18 @@ export const getTask = (
+					'Complete your Professional Email setup to start sending and receiving emails from your custom domain today.'
+				),
+				actionText: translate( 'Complete setup' ),
+				isSkippable: true,
```
I don't think we should allow this to be skippable.

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx:
```diff
@@ -321,6 +326,12 @@ export default connect( ( state ) => {
+	const domain = getSelectedDomain( {
+		domains,
+		selectedDomainName: siteSlug,
+	} );
```
Given that we're picking the domain with `name === siteSlug` and then using only `domain.name` below, I think we can just use `siteSlug` directly.

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +191,18 @@ export const getTask = (
+				),
+				actionText: translate( 'Complete setup' ),
+				isSkippable: true,
+				actionUrl: emailManagement( siteSlug, domainName ),
```
Rather than depending on `domainName` derived from the current `siteSlug`, maybe we should send the domain name from the back end -- I see that's what we're doing for the `CHECKLIST_KNOWN_TASKS.DOMAIN_VERIFIED` task above.

AllTerrainDeveloper replied:
I'll change this to be `task.domainName` but we should be sending it from the backend then as you said 🙂

daledupreez replied:
Given that the back end uses snake_case, I am going to keep this to just `domain`! 🙃

mattwiebe committed:
Compat: change legacy FSE tag to prevent WP 5.8 issues (#54930)

Stop squatting on the `full-site-editing` theme tag for Legacy FSE.

daledupreez reviewed (commented):
I shared some thoughts here.

AllTerrainDeveloper committed:
Emails: Partly apply Pull Request improvements

AllTerrainDeveloper committed:
Emails: Partly apply Pull Request improvements

andres-blanco committed:
Theme Showcase: Fix FSE Beta badge size (#54894)

* better FSE Beta badge

Commit:
Fix issues with Activity Log on Jetpack Multi-Site Secondary Sites (#54978)

flootr committed:
chore: migrate `client/root.js` to `import/order` (#54960)

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +190,18 @@ export const getTask = (
+		case CHECKLIST_KNOWN_TASKS.PROFESSIONAL_EMAIL_MAILBOX_CREATED:
+			taskData = {
+				timing: 2,
+				title: translate( 'Setup your Professional Email' ),
```
Can we make this `Set up your Professional Email`? The key change is splitting "Setup" into two words.
```suggestion
				title: translate( 'Set up your Professional Email' ),
```

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx:
```diff
@@ -339,7 +343,7 @@ export default connect( ( state ) => {
 		isPodcastingSite: !! getSiteOption( state, siteId, 'anchor_podcast' ),
 		menusUrl: getMenusUrl( state, siteId ),
 		siteId,
-		siteSlug: getSiteSlug( state, siteId ),
+		siteSlug,
```
It may be worth just keeping the original code for this file, as we are now introducing a new `siteSlug` variable that's only used once.

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +190,18 @@ export const getTask = (
+				),
+				actionText: translate( 'Complete setup' ),
+				isSkippable: false,
+				actionUrl: emailManagement( siteSlug, task.domainName ),
```
I've now tested the back end using `domain` as the field for the domain.
```suggestion
				actionUrl: emailManagement( siteSlug, task.domain ),
```

AllTerrainDeveloper replied:
Just my thought, but seems to me that domain would be an object with more properties inside, instead of a string. I'm happy to add `task.domain` but for my understanding would be better `task.domainName` or `task.domain.name`

daledupreez replied:
I agree that `domainName` would be better. But you get to make a quick choice between `domain` and `domain_name` for now. (I don't think I can add `domainName` due to linting/style requirements on the back end.)

daledupreez reviewed (commented):
I still have one or two comments, but I think we should be able to test this tomorrow. I think we'll want to delay the back end deploy for at least a day or two to get some of the data cleanup/handling for mailbox events cleaned up, but that shouldn't block getting the client code deployed.

flootr committed:
chore: migrate `packages/load-script` to `import/order` (#54963)

mreishus committed:
Theme Search: Magic welcome click clears previous unfinished attribute searches (#54774)

fullofcaffeine committed:
Remove deprecated GB v11.0 fallback CSS selectors (#54989)

LL782 committed:
Fix minor typo in calypso-build README (#54975)

worldomonation committed:
Playwright: remove migrated Selenium tests. (#54775)

* Remove migrated Selenium tests from repository.

* Also remove theme activate test.

leonardost committed:
Add the "compare_arrows" Material design icon to WPCOM (#54982)

This PR adds the `compare_arrows` icon to Calypso.

This icon will be used in the "Verifying connection" domain status notices in #54649.

renovate[bot] committed:
fix(deps): update dependency tslib to ^2.3.0 (#55002)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

renovate[bot] committed:
fix(deps): update dependency stylelint to ^13.13.1 (#55001)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

jessie-ross committed:
Use authenticated error logging (#54885)

* ExPlat: Send errors authenticated

* Send errors authenticated

deBhal committed:
My Home: Improve Rendering of Multiple Primary Cards (#54367)

* Render primary cards with DotPager component

* DotPager: Allow props

* Add a targeting class for the primary card area

* Add props to DotPager to customize button icons and add text labels

* Change primary buttons to arrows and display text labels

* Styles: move controls up, arrow to the right, shrink dots, remove box-shadow

* Switch to dashicons arrows, tweak DotPager to accept icons as elements instead of strings, drop alignment styles

* Switch current dot color to gray 80

* Set padding for primary task to 32px and align contents against that

* Color & whitespace css fine-tuning

* Move card text padding onto DotPager controls so they collapse properly

* Add some comments and whitespace to the css

* Round the border corners

* 8px margin for the arrows

* The last few pixels to align the content edges vertically inside the card

* Fix disabled button color

* Use flex gap to simplify the spacing style

* Add extra primary cards with dev query param

* restore button labels

* integrate & fix styles after the rebase

* Fix controls margin-bottom

* whitespace

* Correct import order (style fix)

* Improve label+icon alignment for dotpager control

* s/customer-home__location-primary/primary__customer-home-location-content/ hat-tip @arthur791004

* drop unnecessary styles (hat-tip @arthur791004)

* clarify flag to show text labels for dot-pager component

* non-semantic tidying of css

* update DotPager prop in my-home

* Simplify and reduce nesting in DotPager styles

* drop query param check

* Remove unused prop from CustomerHome

* Kill negative padding and restore control of outer whitespace to children

* Remove additional lines in mobile view

* Remove additional lines in mobile view take 2

* Eliminate extra whitespace when the checklist switches to one-column

* clean up commented out styles

* Align site-setup-list grid with overall my-home grid

flootr committed:
chore: migrate `packages/onboarding` to `import/order` (#54964)

flootr committed:
chore: migrate `client/gutenberg` to `import/order` (#54966)

flootr committed:
chore: migrate `packages/calypso-analytics` to `import/order` (#54968)

flootr committed:
chore: migrate `packages/social-previews` to `import/order` (#54969)

scinos committed:
chore: Migrate packages/wpcom.js to use import/order (#55000)

scinos committed:
chores: Fixes teamcity version (#55010)

flootr committed:
chore: migrate `packages/state-utils` to `import/order` (#54970)

scinos committed:
chore: Uses the new credentialsJSON token for e2e encryption key (#55013)

flootr committed:
chore: migrate `packages/wp-babel-makepot` to `import/order` (#54971)

* chore: migrate `packages/wp-babel-makepot` to `import/order`

* tests: update jest snapshot

flootr committed:
chore: migrate `client/document` to `import/order` (#55007)

flootr committed:
chore: migrate `packages/page-pattern-modal` to `import/order` (#55008)

AllTerrainDeveloper committed:
Emails: Pull Request suggestions

AllTerrainDeveloper committed:
Emails: Pull Request suggestions

flootr committed:
chore: migrate `packages/i18n-calypso` to `import/order` (#55011)

flootr committed:
chore: migrate `packages/calypso-build` to `import/order` (#55009)

flootr committed:
chore: migrate `packages/calypso-codemods` to `import/order` (#55012)

* chore: migrate `packages/calypso-codemods` to `import/order`

* tests: update jest snapshot

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -189,6 +190,18 @@ export const getTask = (
+				),
+				actionText: translate( 'Complete setup' ),
+				isSkippable: false,
+				actionUrl: emailManagement( siteSlug, task.domain ),
```
This should link to the new URL being added in PR #54934, which will be behind the `emailManagementCompleteTitanMailboxSetup()` path from `my-sites/email/paths.js`.
```suggestion
				actionUrl: emailManagementCompleteTitanMailboxSetup( siteSlug, task.domain ),
```

daledupreez commented on client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js:
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import InlineSupportLink from 'calypso/components/inline-support-link';
 import { localizeUrl } from 'calypso/lib/i18n-utils';
 import { domainManagementEdit, domainManagementList } from 'calypso/my-sites/domains/paths';
+import { emailManagement } from 'calypso/my-sites/email/paths';
```
As noted below, this should use the new path helper being added in PR #54934.
```suggestion
import { emailManagementCompleteTitanMailboxSetup } from 'calypso/my-sites/email/paths';
```

daledupreez reviewed (commented):
Other than the path change, I think this is looking good!

Once #54934 gets merged, let's update the path helper and get this shipped! Could you please change this from a draft PR to a "real" PR in the meantime?
 
I think it's worth explicitly noting that D64662-code is needed on the back end to provide the task data, as well as noting that you can force the setup checklist to appear for a site by navigating to `/home/:siteSlug?view=view_site_setup&dev=true`.

Finally, could you update the screenshots to reflect the updated strings that we're using?

flootr committed:
chore: migrate `client/data` to `import/order` (#55022)

arcangelini committed:
Add conditional logic to insertHtmlTag function (#54999)

david-szabo97 committed:
Unified Sidebar: Add badge (#54681)

Commit:
Add money back mention in checkout page (#54169)

* Add money back mention in checkout page

* Move checkout footer to wpcom codebase

* Show money back mention only for yearly Jetpack products

* Update composite checkout README

* Add 7 day money back guarantee mention

* Handle case both monthly and yearly products in cart

* Fix checkout page hidden by footer

* Fix TS errors

* Remove unnecessary deps from React effects

* Update checkout page margin bottom on resize

flootr committed:
chore: migrate `packages/calypso-url` to `import/order` (#55015)

flootr committed:
chore: migrate `client/login` to `import/order` (#55018)

worldomonation committed:
Playwright: migrate `notifications` spec. (#54848)

* Implement notification trash spec.
- add new components to represent notification panel and notifications.
- updates to LoginFlow and CommentsComponent to improve reliability.

* Wait for `networkidle` prior to opening the notifications panel.

* Use `page.waitForURL` when logging in to prevent `page.goto` from preemptively navigating away

* Move the `waitForURL` call to the test step.

* Make `CommentsComponent.postComment` wait until the text area is stable.

* Wait for URL and notifications button to be fully loaded before attempting to open notifications.

* Add comments on CommentComponent explaining reasons for using waitForElementState.
Reorganize `NavbarComponent.openNotificationsPanel` to wait for the notification button to be stable.

* Use updated method name `clearAuthenticationState`

* Code review comments.
- `useKeyboard` instead of `keyboard`.
- export `config` object.

chrisfromthelc committed:
Checkout: Improve post-renewal copy and fix "learn more" link (#54993)

* Updated notification text and link to support doc

* Replce old renewal message with one similar to non-autorenew plan renewals.

* Adjust link text to be explicit about the link destination.

* Removed concatenation and unused argument from autonrenew notice.

* Remove concatenation from non-auto-renew notification.

worldomonation committed:
Playwright: migrate coblocks tests. (#54337)

* Start new branch to migrate coblock tests.

Include test file

Parametrize additional tests.

* - address eslint.
- migrate the WPCOM gutter test.
- add new method `setGutter` as member method of PricingTableBlock.

- parallelize each coblock test.
- new function DataHelper.createSubsuiteTitle to generate formatted subsuite test name.

Linter fixes.
- fill JSDoc for new methods.
- remove use of `any` within inline functions.

Large rewrite of the coblock tests.
- split the twitter tests from rest due to an existing issue that auto-dismisses the publish panel.
- moved gutter tests to its own file.
- implemented files for each block.
- implemented a static method to check on the published page whether the block is present.

* Rebase and use Jest.

* Update the expected CSS for Dyanmic HR block.
Update JSDoc for return value on block classes.

* Placate the linter

* PR comments.
- use `waitForFunction` when setting gutters.
- fix typo in GutenbergEditorPage.
- remove header from BaseBlock class.
- simplify column selection in PricingTableBlock.
- remove unnecessary page objects from all coblock tests.

* Major changes to address code review comments:
- use static property in the Block classes in test specs.
- static `validatePublishedContent` method to check for content entered into the block instead of asserting on wrapper classes.
- merged Click to Tweet test into the main coblock test suite.

* Define gutter values in types.

* Add proper mobile support.

* Address review comments.
- modify some comments in code.
- move type definition for gutterValues to the PricingTableBlock itself.
- parametrize the gutter value check using values defined in PricingTableBlock.

* Remove inheritance from blocks

* Remove BaseBlock file.

mattgawarecki committed:
Activity Log: Convert ActivityCard component to functional+TypeScript (#54973)

fullofcaffeine committed:
DRY the `insertBlockOrPatternViaBlockAppender` function by extracting it to an `AbstractEditorComponent` class (#54997)

In this case, inheritance was simpler than composing it, due to the two existing editor classes;

- The `GutenbergEditorComponent` and `SiteEditorComponent` now inherit from this abstract class;
- The class provides a noop `runInCanvas` instance method and lets subclasses override the implementation;
- The abstract class can be used to DRY other functions, too, later.

blackjackkent committed:
Fix logic that checks if recommended/trending tab are loading (#54979)

blackjackkent committed:
Display FSE theme tab when current enabled theme supports FSE (#55043)

worldomonation committed:
Update copy on the Verblio card. (#54699)

rcanepa committed:
Jetpack site-less Checkout: update ZD ticket after user schedules a call (#55036)

* Add util to update ZD ticket

* Add listener to react to an schedule event

deBhal committed:
DotPager: Ignore non-current pages when sizing DotPager (#55006)

* Ignore non-current pages when sizing DotPager

* Create a containing stacking context to clip content as it shifts away

scinos committed:
chore: Prepare for releasing @automattic/calypso-live 9.0.0 (#55004)

arthur791004 committed:
My Home: Update the copy in the site setup checklist (#54749)

* Update the copy in the site setup checklist

* Revert unnecessary changes in current stage

p-jackson committed:
My Home: Learn & Grow section ignores cards it doesn't recognise (#55051)

worldomonation committed:
Playwright: extract TeamCity configuration for Playwright to a function (#55047)

* Extract common setup steps to a function for Playwright tasks.

* Include UUID for mobile Playwright.

Address review comments from Sergio.

* Enable Playwright for all branches excluding `pull` for both mobile and desktop.

* A little too fast...
Remove the parameter `filter` from function `playwrightBuildType`.

Commit:
Domains: Add Professional Email upsell route (#54941)

* Add route and basic layout for email upsell page
* Add upsell path helper function
* Move upsell path handler definition
* Reorder imports and definitions
* Replace placeholder domain with domain prop
* Use domainAddNew helper to get path
* Use Fragment shorthand syntax
* Change import order of EmailProvidersUpsell
* Rename controller function to be more specific
* Inline translateArgs value
* Fix: registerDomain -> domain
* Reorder, use path alias for domainAddNew import

daledupreez committed:
Enable the signup/professional-email-step flag in development and wpcalypso (#55027)

flootr committed:
chore: migrate `apps/o2-blocks` to `import/order` (#55024)

david-szabo97 committed:
Theme showcase: Add beta badge to FSE themes (#54706)

olaseni committed:
Provision available mailboxes for Professional Email (#54934)

* Add a new page and controller bits to create a mailbox
* Reformat imports
* Rebase
* Rebase
* Documentation for queries and mutations
* Decorate components with Proptypes
* Accept to use existing string
* Fix variable rename missed from a previous refactor
* Dispatch event for alternate scenario
* Copy correction
* Refine error display for mailbox availability
* Refine prop name `showAddAnotherMailboxButton`
* Tweak button CTA to `Set Up`
* Refine HeaderCake text
* Refine HeaderCake text
* Fix js doc and back button spec
* Working on validation
* Refine validation logic
* Refine retry sematics for availability
* Rename flag, and redirect to the Email Upsell page when there's no Titan subscription

delputnam committed:
Add "timer" icon (#55056)

AllTerrainDeveloper committed:
Merge branch 'implement/professional-email-setup-card' of github.com:Automattic/wp-calypso into implement/professional-email-setup-card

daledupreez commented:
After the merge noise, we moved the actual changes to PR #55061 to simplify review.
